.github/workflows/codecheck.yml

```diff
@@ -17,3 +17,5 @@ jobs:
         run: yarn
       - name: Check Typescript syntax
         run: yarn codecheck
+      - name: Find dead code
+        run: yarn find-deadcode
```

Comment:
*Maybe* do this in a separate Workflow?

Reply:
We should avoid to make a new workflow for each small checker.
I think the limits are 6 workflows in parallel. We would waste time here to setup a ubuntu server and install dependencies.

Reply:
Ok 20 is the limit, still i would not cluster the checker with to much workflows.

Reply:
Added to linter workflow